### PR TITLE
711 Exclude `nsfw` companies from "related companies"

### PR DESCRIPTION
**Pull request by Torfab**
Solved #711 exclude nsfw from related companies

### CODE REVIEW

File: layouts/partials/related-companies.html [hunk 1]
```diff
@@ -9,8 +9,10 @@
         {{ $num_common_categories := intersect $.Params.categories .Params.categories | len }}
         {{ $num_common_countries := intersect ($.Param "relevant-countries") (.Param "relevant-countries") | len }}
         {{ if and (gt $num_common_categories 0) (ne .Permalink $.Permalink) }}
-            {{ $score := add (div (float $num_common_categories) $num_categories) (div $num_common_countries $num_countries) }}
-            {{ $related_pages = $related_pages | append (dict "page" . "score" $score) }}
+            {{if not .Params.nsfw}}
```
Comment (baltpeter) [inline layouts/partials/related-companies.html:12]: I think it would be cleaner to add this to the outer `if`.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the PR! I have left some comments below.

File: layouts/partials/related-companies.html (hunk 1 above)
Comment (Torfab) [reply layouts/partials/related-companies.html]: sure, done.

File: layouts/partials/related-companies.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
     {{ range (where .Site.Pages "Section" "company") }}
         {{ $num_common_categories := intersect $.Params.categories .Params.categories | len }}
         {{ $num_common_countries := intersect ($.Param "relevant-countries") (.Param "relevant-countries") | len }}
-        {{ if and (gt $num_common_categories 0) (ne .Permalink $.Permalink) }}
-            {{ $score := add (div (float $num_common_categories) $num_categories) (div $num_common_countries $num_countries) }}
+        {{ if and (gt $num_common_categories 0) (ne .Permalink $.Permalink) (not .Params.nsfw)}}
+            {{ $score := add (div (float $num_common_categories) $num_categories) ((div $num_common_countries $num_countries)) }}
```
Comment (baltpeter) [inline layouts/partials/related-companies.html:12]: Why the double parentheses?

Comment (Torfab) [reply layouts/partials/related-companies.html]: typo, solved

**Review by baltpeter — APPROVED** [review]
Thank you.

**Pull request merged**